### PR TITLE
Jv hw dao seven

### PR DESCRIPTION
**Pull request by Sashk52**

### CODE REVIEW

File: src/main/webapp/WEB-INF/views/index.jsp [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: Tsven
+  Date: 20.01.2021
+  Time: 18:29
+  To change this template use File | Settings | File Templates.
+--%>
```
Comment (semyonich) [inline src/main/webapp/WEB-INF/views/index.jsp]: Remove autogenerated comments from index.jsp. 
Before submitting solution make sure you checked it first with checklist

File: src/main/webapp/web.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+    <servlet>
+        <servlet-name>index</servlet-name>
+        <servlet-class>taxi.controllers.IndexController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>index</servlet-name>
+        <url-pattern>/index</url-pattern>
+    </servlet-mapping>
+</web-app>
```
Comment (semyonich) [inline src/main/webapp/web.xml]: add new line at the end of the file

**Review by semyonich — CHANGES_REQUESTED** [review]

File: src/main/webapp/web.xml (hunk 2 above)
Comment (Sashk52) [reply src/main/webapp/web.xml]: done

File: src/main/webapp/WEB-INF/views/index.jsp (hunk 1 above)
Comment (Sashk52) [reply src/main/webapp/WEB-INF/views/index.jsp]: done

**Review by semyonich — APPROVED** [review]

File: src/main/java/taxi/service/CarService.java [hunk 3]
```diff
@@ -1,22 +1,11 @@
...
-    Car create(Car car);
-
-    Car get(Long id);
-
-    List<Car> getAll();
-
-    Car update(Car car);
-
-    boolean delete(Long id);
+public interface CarService extends GenericService<Car, Long> {
```
Comment (KseniiaMakarova) [inline src/main/java/taxi/service/CarService.java:7]:
```suggestion
public interface CarService extends GenericService<Car, Long> {
    List<Car> getAllByDriver(Long driverId);
```

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**